main.go

```diff
@@ -26,6 +26,10 @@ import (
 var BuildID string
 
 func main() {
+	appEnv := os.Getenv("APP_ENV")
+	if len(appEnv) > 0 && (appEnv == "dev" || appEnv != "development") {
```

Comment:
did not understand ? why is this needed ? also, condition looks wrong. check once

Reply:
For local running, I need to pass data from .env file